dotnet/src/Functions/Functions.OpenApi/Extensions/OpenApiKernelExtensions.cs

```diff
@@ -258,7 +258,7 @@ public static class OpenApiKernelExtensions
     /// <param name="documentUri">The URI of OpenAPI document.</param>
     /// <param name="loggerFactory">The logger factory.</param>
     /// <returns>An instance of <see cref="KernelFunctionFromPrompt"/> class.</returns>
-    private static KernelFunction CreateRestApiFunction(
+    internal static KernelFunction CreateRestApiFunction(
```

Comment:
@SergeyMenshykh, I am not sure if you prefer these to be `public` or `internal` (and exposed to `Functions.OpenApi.Extensions` project via `InternalsVisibleTo` in `csproj`). I went with the latter, but please advise if you prefer a different approach.

Reply:
Let's go with the internal option for now; we can revise it later if needed.